src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpIntelliSense.cs

```diff
@@ -36,7 +35,7 @@ public override async Task InitializeAsync()
             VisualStudio.Workspace.SetImportCompletionOption(false);
         }
 
-        [WpfTheory, CombinatorialData]
+        [WpfTheory(Skip = "https://github.com/dotnet/roslyn/issues/70582"), CombinatorialData]
```

Comment:
>[WpfTheory(Skip = "https://github.com/dotnet/roslyn/issues/70582"), CombinatorialData] [](http://example.com/codeflow?start=8&length=86)

Is Gen not making a PR with his commit of this? ([https://github.com/genlu/roslyn/commit/492d12fe051dc8a07d0a982d191a484e892eedc1](https://github.com/genlu/roslyn/commit/492d12fe051dc8a07d0a982d191a484e892eedc1))

Reply:
he is.  this is just unblocking whichever pr goes in first.

Reply:
(this is a merge of his branch).